src/index.js

```diff
@@ -202,7 +197,6 @@ export default {
           cwd,
           stdin,
           stream: 'both',
-          timeout: this.linterTimeout,
```

Comment:
The people running into the default timeout were having the issue because they were running Rubocop with JRuby, which has a horribly slow spinup time and is completely inappropriate for Rubocop. See https://github.com/AtomLinter/linter-rubocop/issues/202.